data/com.luxoft.ConnectivityManager.xml

```diff
@@ -4,6 +4,11 @@
 <!-- TODO: Note that API should not be considered stable yet and is still
      subject to change. Should go through all TODOs before removing this note. -->
 <node>
+  <!--
+      Main interface for connectivity-manager.
+
+      Goal with API is to make it easy to implement graphical user interfaces.
```

Comment:
If you have any ideas for how to reword/improve this, please come with suggestions. :)

Reply:
Sure, not easy but here is a proposal :).
Used to control and retrieve information of all supported technologies, e.g. WiFi connectivity.

API goals are easy-of-use and provide unique functionality to be used in an automotive context.

Reply:
@JEderonn Hum, reading your comment about adding something in the XML file again... I am not sure what I added is what you were after.

Reply:
Sounds better.